### ISSUE TITLE
Fresh CL install doesn't boot

### ISSUE THREAD

**Issue by Aqua1ung** [clearlinux/distribution#2920]
[There seems to be an issue](https://community.clearlinux.org/t/failed-boot-after-install/8953) with recent CL desktop releases (haven't tried the server myself) and **recent BIOSes**, whereby freshly installed CL does not boot. The issue can be circumvented by installing a very old CL release (37860 worked for me), and then updating to the latest (`swupd update`). Clearly, then, recent CL releases are to blame, as freshly-installed pre-37860 releases boot just fine.

**Comment by Aqua1ung** [comment]
[Workaround](https://community.clearlinux.org/t/install-linux-clear-lost/8958/13?u=aqua1ung): it appears that the installation _requires_ the `x11-tools` bundle or else the freshly installed CL won't boot into the GUI, so either add this bundle during installation from the list of optional bundles, or add it via ssh afterward:
```
sudo swupd bundle-add x11-tools
sudo reboot
```
@fenrus75, is this a bug, or a feature? And if it's the former, would you mind fixing it please?

**Comment by OpenSauce04** [comment]
+1, fresh install is also borked on my end after being previously successful when installing Clear Linux on the same hardware a few months ago

**Comment by asreich** [comment]
I have been screwing with this for the past week actually and can tell you from my testing that 37980 is the last one that works for me. I haven't tested everything between then and now but I basically binary searched and tested versions between 37860 and 39590 to try and narrow it down to a specific window. I tried .iso where I could but many were missing.

Full list of tests:

37860 works
37940 works
37980 works
37990 did not test 
38000.img.xz does not work
38270.img.xz does not work
38990.iso does not work
39400.iso does not work
39590.iso does not work

I believe I remember reading all installations after 38270 are Wayland based, but that doesn't match exactly with when this broke. I tried reading through the RELEASE NOTES that are published along with each release but it's all just package bumps and I'm not sure where to go to even read through specific commits. If you have a link I'm happy to go diving.

Not sure it makes sense to test any further ISOs if we know the fix is coming, but unless someone knows otherwise, I believe all/most installer releases between about 37980 and whenever the fix is merged are broken? This feels like it can't be true because SOMEONE would have had to notice it by now but maybe there aren't that many people installing ClearLinux regularly?
